src/os/README.md

```diff
@@ -50,19 +50,23 @@ inspiration for several decisions in this library.
 - do not use `OS\ErrnoException` if the error condition would not be indicated
   by the `errno` variable in C. Consider adding another similar class, e.g.
   add `OS\HErrnoException` if you want to report an error exposed via `h_errno`
-- add and use Hack classes (not type aliases) for 'handle'-like parameters and
-  return values, e.g. `OS\open()` returns a `HH\Lib\FileDescriptor` instead of
-  an `int`; as well as aiding type safety, this prevents requests from interfering
-  with resources that belong to another request.
+- use `keyset<Flag>` to represent a C bit set, where `Flag` is a Hack `enum`.
```

Comment:
I don't think this has been discussed, and would contract the existing behavior

Reply:
I split this item into another PR. https://github.com/hhvm/hsl/pull/171

Let's discuss it over there.

Reply:
I just wanted to settle down the `shape` / `vec` encoding things.